infrastructure/ArmTemplates/DeployAllTemplate.json

```diff
@@ -103,105 +96,128 @@
                     "siteName": "[replace(parameters('RuntimeHost1'),'1','2')]"
                 }
             ]
+        },
+        "RedisCache": {
+            "Name": "DiagnosticsCache-Prod",
+            "diagnosticsEnabled": true,
+            "existingDiagnosticsStorageAccountId": "/subscriptions/0e7b3183-4428-4c1f-93cd-cb2a26616a8a/resourceGroups/RedisCache-RG/providers/Microsoft.Storage/storageAccounts/storeokauaj64yhhns"
```

Comment:
Possible to not checkin the exact resource uri here?

Reply:
I'm not sure about that. I'm not even sure this template runs. We could probably delete this PR.